File: JavaToC++/ex2.java
//This line is a comment
//Header information like below goes at the top of your programs

/*
Class: CPSC 122
Section: 01
Team Member 1: Paul De Palma
Team Member 2: None
Submitted By: Paul De Palma
GU Username: depalma
File Name: ex2.java
Program illustrates fundmental data types 
To Build: javac ex2.java
To Execute: java ex2 
*/

import java.lang.Math;

public class ex2 
{
  public static void main(String[] args)
  {
   int distance_to_sun = 94359000;
   double sun_E_notation = 9.4359E7;
   String str = "Various Items";
   System.out.println("Here are " + str);
   System.out.println("Distance to the sun: " + distance_to_sun + " miles"); 
   System.out.println("Distance to the sun: " + sun_E_notation + " miles"); 
   
   double pi = 3.14156;
   double radius = 2.0;
   double area;
   area = pi * radius * radius;
   System.out.println(area);
   area = pi * Math.pow(radius,2.0);
   System.out.println(area);


   char ch = 'A';
   ch++;
   System.out.println(ch);
   System.out.println(ch + 1);

   boolean cs_is_fun = true;
   if (cs_is_fun)
     System.out.println("Computer Science is Fun");

  }
} 
	

File: A-JavaToC++/ex2.cpp

/*
Name : Paul De Palma
Class: CPSC 122, Section 1
Date: January 9, 2022
Assignment: Example 2
Description: Program illustrates fundmental data types and a simple if statement 
Notes: See Gaddis, Section for 3.7 for a more complete discussion of formatting
*/

#include <iostream>
#include <math.h>  //necessary for pow(base,exp), below
#include <iomanip> //necessary for setprecision
using namespace std;

const double DISTANCE_TO_SUN = 94359000;
const double DISTANCE_TO_SUN_E = 9.4359E7; 
const double PI = 3.14156;
const double RADIUS = 2.0;

int main()
  {
   string str = "Various Items"; 
   cout << "Here are " << str << '\n';
   cout << "Distance to the sun: " <<  DISTANCE_TO_SUN_E << " miles" << endl; 
   cout.setf(ios::fixed);
   cout <<  setprecision(0);
   cout <<  "Distance to the sun: " <<  DISTANCE_TO_SUN <<  " miles" << endl;

   double area;
   cout <<  setprecision(5);
   area = PI * RADIUS * RADIUS;
   cout << area << endl;
   area = PI * pow(RADIUS,2.0); //pow(base,exponent)
   cout << area << endl;

   char ch = 'A';
   ch++;  //a character is an 8 bit int 
   cout << ch << endl;
   cout << ch + 1 << endl; //as this output shows

   bool cs_is_fun = true;
   if (cs_is_fun)
     cout << "Computer Science is Fun" << endl;

   return 0;

}
	
